### DISCUSSION TITLE
[Question]: Exclude cameras from filmstrip in 0.14

### DISCUSSION THREAD

**Discussion by JoFranke** [blakeblackshear/frigate#12699] [Ask A Question] [answered]
### What is your question:

I would like not to show events from some cameras at the new filmstrip. Reason is, some cams generate events every minute, which is wanted but I am not interested in the filmstrip - is that possible? Thanks.

**Comment by hawkeye217** [comment, marked as answer]
The filmstrip only shows recent alerts, so the best way to remove cameras from it would be to have events on those cameras classified as detections instead of alerts. 

The docs lay out several ways of accomplishing this. 

Creating a camera group without the event generating camera will also exclude its events from the filmstrip.

**Comment by JoFranke** [comment]
Thanks a lot, I will try that
